mantis-server/mantis-server-worker/src/main/java/io/mantisrx/server/worker/jobmaster/AutoScaleMetricsConfig.java

```diff
@@ -17,18 +17,28 @@
 package io.mantisrx.server.worker.jobmaster;
 
 import static io.mantisrx.server.core.stats.MetricStringConstants.*;
+import static io.reactivex.mantis.network.push.PushServerSse.DROPPED_COUNTER_METRIC_NAME;
 
 import java.util.HashMap;
+import java.util.HashSet;
 import java.util.Map;
 import java.util.Set;
+import java.util.regex.Pattern;
 
 
 public class AutoScaleMetricsConfig {
 
+    public static final String CLIENT_ID_TOKEN = "_CLIENT_ID_";
+    public static final String OUTBOUND_METRIC_GROUP_PATTERN = "PushServerSse:clientId=" + CLIENT_ID_TOKEN + ":*";
```

Comment:
suggest using a string constant for `PushServerSse` / `ServerSentEventRequestHandler` and `clientId` that is used for both metric generation and consumption here. So we don't unintentionally break this if/when the metric name is changed or a new tag is added to the metric etc.